axelar-chains-config/info/testnet.json

```
@@ -3246,6 +3246,96 @@
           "implementation": "0x6977477Bd4C053E1BfD0B6F287060c2A4E5E1aA1"
         }
       }
+    },
+    "memento-demo": {
+      "name": "Memento Demo",
```

```suggestion
      "name": "Memento",
```
display name should be nicer